packages/slice-machine/src/modules/selectedCustomType/sagas.ts

```diff
@@ -30,6 +25,11 @@ export function* saveCustomTypeSaga() {
     }
 
     yield call(saveCustomType, currentCustomType, currentMockConfig);
+    void Tracker.get().trackCustomTypeSaved({
```

Comment:
Interesting choice to put the tracking.

- Was it tested ?
- What happens when saving the custom type fails ?
- Why did you choose to put the tracking here instead of after a direct action ?

I'm not saying one approach is better than the other, i'd like for us to set a rule to follow from now on 😄.
Because at the moment, all the events are located with the code itself, not in the saga yet.
That's an interesting debate.

Reply:
duplicated